boards/arm/xiao_ble/doc/index.rst

```diff
@@ -1,14 +1,14 @@
 .. _xiao_ble:
 
-XIAO BLE
-########
+XIAO BLE (SENSE)
```

Comment:
You mean just to replace `SENSE` with `Sense`, right?

Reply:
Changed `SENSE` -> `Sense`, to make it consistent with https://wiki.seeedstudio.com/XIAO_BLE/. However I noticed that the "officially used" name is now `XIAO nRF52840` instead of `XIAO BLE`, which is true for wiki and seeed online shop. If we decide to change that, it should be part of another PR. Please notice that the link to wiki still contains `XIAO BLE` in the URL path.